packages/js/src/utils/id-token.ts

```diff
@@ -0,0 +1,39 @@
+import { UrlSafeBase64 } from '@silverhand/essentials';
+import * as s from 'superstruct';
+
+/**
+ * @link [ID Token](https://openid.net/specs/openid-connect-core-1_0.html#IDToken)
+ */
+const IdTokenClaimsSchema = s.type({
+  iss: s.string(),
+  sub: s.string(),
+  aud: s.string(),
+  exp: s.number(),
+  iat: s.number(),
+  at_hash: s.optional(s.string()),
+});
+
+export type IdTokenClaims = s.Infer<typeof IdTokenClaimsSchema>;
+
+export const decodeIdToken = (token: string): IdTokenClaims => {
```

Comment:
Can we use the jose lib to decode ID token like @gao-sun mentioned in this [PR](https://github.com/logto-io/kotlin/pull/74#discussion_r787345337)?

Reply:
@IceHe tested, unfortunately jose cannot do the job.